src/spyglass/decoding/decoding_merge.py

```diff
@@ -1,29 +1,54 @@
+from itertools import chain
+from pathlib import Path
+
 import datajoint as dj
-from spyglass.utils.dj_merge_tables import _Merge
-from spyglass.decoding.v1.clusterless import ClusterlessDecodingV1
-from spyglass.decoding.v1.sorted_spikes import SortedSpikesDecodingV1
+
+from spyglass.decoding.v1.clusterless import ClusterlessDecodingV1  # noqa: F401
+from spyglass.decoding.v1.sorted_spikes import (
+    SortedSpikesDecodingV1,
+)  # noqa: F401
+from spyglass.settings import config
+from spyglass.utils import SpyglassMixin, _Merge, logger
 
 schema = dj.schema("decoding_merge")
 
 
 @schema
-class DecodingOutput(_Merge):
+class DecodingOutput(_Merge, SpyglassMixin):
     definition = """
     merge_id: uuid
     ---
     source: varchar(32)
     """
 
-    class ClusterlessDecodingV1(dj.Part):
+    class ClusterlessDecodingV1(SpyglassMixin, dj.Part):
```

Comment:
Was a V0 part table intentionally excluded?

Reply:
Yes, because V0 isn't really a pipeline with a end table. It was more of a placeholder.